lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -392,14 +392,14 @@ TrajectoryParams LCIStrategicPlugin::handleFailureCaseHelper(const lanelet::Carm
   traj_upper.t0_ = current_time;
   traj_upper.v0_ = starting_speed;
   traj_upper.x0_ = starting_downtrack;
-  traj_upper.is_algorithm_successful = false;
+  traj_upper.is_algorithm_successful = true;
```

Comment:
I think the better way would be setting it to false at first and changing it to true if the trajectory is created.

Reply:
Trajectory is created in lci tactical plugin. Strategic plugin calls tactical so it needs to be true here.

Reply:
I know. But it shall remain false unless we have a valid trajectory. So we should keep it false and change it to true in any condition that we successfully create a trajectory. This way, we would know if we do not have a successful trajectory.

Reply:
Trajectory is created in a totally different plugin (tactical). And we have to make it true here otherwise that tactical plugin will ignore all this change as it will be running last successful trajectory. So we have to make it true here.
Having it false will make the tactical plugin ignore the trajectory. So it needs to be true either way.

Reply:
I understand that if it be false, then the tactical plugin does not create any trajectory. But what I am saying is that we should not assume this is true before finding the upper trajectory. I agree that we will most likely create a successful one every time, but this parameter indicates whether we are able to fill out the trajectory info correctly or not. Anyway, not a big issue. I leave it to you.

Reply:
If its not successful trajectory then we are not using these parameters.

Reply:
I believe there is a confusion here. Please let me know if you need to talk over phone to clear this up. But you can skip this for now.